apps/legal_archive/resource.py

```diff
@@ -66,4 +66,4 @@ class LegalPublishQueueResource(LegalResource, PublishQueueResource):
     item_schema.update(PublishQueueResource.schema)
     schema = item_schema
 
-    datasource = {'source': LEGAL_PUBLISH_QUEUE_NAME}
+    datasource = {'source': LEGAL_PUBLISH_QUEUE_NAME} # type: ignore
```

Comment:
Why is `datasource` requiring `type: ignore` here but none of the other classes require it?

Reply:
`PublishQueueResource` redefines type defined in original `Resource.datasource`.